app/assets/stylesheets/layouts/l-data-portal-country.scss

```diff
@@ -227,7 +227,6 @@
         display: inline-block;
         min-height: 71px;
         max-height: 100px;
-        width: 100%;
```

Comment:
Could you try? I think it could be a bit better approach. What do you think?
```
width: auto;
max-width: 100%;
min-width: 300px;
```

Reply:
if I leave min-width: 300px some of the logos are still deformed